CHANGELOG.md

```diff
@@ -4,10 +4,44 @@ will be documented here.
 
 ## Unreleased
 
+## v0.2.0 TBD
+
```

Comment:
I think this a good point to have a paragraph about major changes, like the improved substrate compatibility.

Reply:
ACK, I'm gonna write something